src/effects/chains/quickeffectchain.h

```diff
@@ -24,4 +24,10 @@ class QuickEffectChain : public PerGroupEffectChain {
 
     void loadChainPreset(EffectChainPresetPointer pPreset) override;
     int numPresets() const override;
+
+  private:
+    /// Keeps track of whether or not a preset has already been loaded. The
+    /// first time a preset is loaded the adopt super knob position setting
+    /// needs to be ignored or its initial value will always be set to 0.0.
+    bool m_hasLoadedPreset = false;
```

Comment:
The member is now unused as well. please remove along with the assignment in `loadChainPreset` and push a fixup again. also feel free to do the rebase squash now. Thank you :+1:

Reply:
Ah thanks, I didn't even compile this normally since I just rebuilt my `mixxx-git` package, so I also missed any warnings. :sweat_smile: